src/manager.tsx

```diff
@@ -1,22 +1,40 @@
 import ReactDOM from "react-dom";
 import React from "react";
 import { App } from "./App";
```

Comment:
let's use React.lazy to load App asynchronously. Otherwise it's always loaded, independently, whether it gets mounted or not.

Reply:
SGTM!